includes/admin/class.llms.admin.post-types.php

```diff
@@ -70,15 +70,6 @@ public function use_block_editor_for_post( $use_block_editor, $post ) {
 
 	}
 
-	/**
-	 * Initializes core for metaboxes
-	 *
-	 * @return void
-	 */
-	public function meta_metabox_init() {
-		include_once 'llms.class.admin.metabox.php';
-	}
-
```

Comment:
This should be deprecated in favor of removed outright to bar against the fatal error from 3rd parties... Though unlikely as it is that this is being used externally.